test/specs/commands/createTransactionCastVotes.js

```diff
@@ -26,7 +26,7 @@ describe('create transaction cast votes', () => {
 				Given('a second passphrase "fame spoil quiz garbage mirror envelope island rapid lend year bike adapt"', given.aSecondPassphrase, () => {
 					Given('a Lisk object that can create transactions', given.aLiskObjectThatCanCreateTransactions, () => {
 						Given('the passphrase can be retrieved from its source', given.thePassphraseCanBeRetrievedFromItsSource, () => {
-							Given('public keys that should be voted for "1234567812345678123456781234567812345678123456781234567812345678"', given.publicKeysThatShouldBeVotedFor, () => {
+							Given('a public key that should be voted for "1234567812345678123456781234567812345678123456781234567812345678"', given.publicKeysThatShouldBeVotedFor, () => {
```

Comment:
Since the description and function name diverge here, prefer a new step definition function or alias.